index.html

```diff
@@ -217,6 +217,32 @@
 No such flexibility is possible when it comes to legal identity.
 
 
+### Tracking Inherent to Credential Formats
```

Comment:
Formats or protocols? (Or ceremonies, oh my!) (That last is a joke; I don't think ceremonies are plausibly associated with this section.)